server.go

```diff
@@ -115,9 +106,11 @@ func main() {
 	histogram := prometheus.NewHistogramVec(prometheus.HistogramOpts{
 		Name: "gitlab_crucible_bridge_request_duration_seconds",
 		Help: "Duration of HTTP requests in seconds",
-	}, []string{"status"})
+		Buckets: prometheus.ExponentialBuckets(0.1, 3, 4),
```

Comment:
I'm not sure if this are the best values. It give 0.1, 0.3, 0.9 and 2.7 buckets. Maybe 0.05, 0.2, 0.8, 3.2 will be better? @Dbzman What do you think?

Reply:
From our experience I can see that most of the requests are in a range of 120 to 200 ms. Some slower ones are around 600 ms. I couldn't see one so far which is below 100 ms. I'm honestly not sure as well which values are the best but I think this is a good starting point. I would merge it now and if it turns out it can be improved, we can adjust the values. :)